src/assets/i18n/de_DE.json

```diff
@@ -781,6 +792,7 @@
   "customer.credentials.passwordreset.invalid_password.error.PasswordExpressionViolation": "Das Kennwort muss mindestens 7 Zeichen enthalten, bestehend aus Zahlen und Buchstaben, ohne Leerzeichen.",
   "customer.missing_fields.error": "Fehlende Felder",
   "customer.wishlist.profane_word.error": "Ihre aktuelle Eingabe kann nicht bearbeitet werden. Unpassende, beleidigende bzw. vulgäre Sprache wird nicht akzeptiert. Bitte geben Sie etwas anderes ein.",
+  "date_format": "dd.MM.yyyy",
```

Comment:
That should not be part of the translation file, as it is part of the Angular localization (sadly, internal API). Get it with:

```ts
import { ɵfindLocaleData } from '@angular/core';

const localeData = ɵfindLocaleData(locale);
return localeData[10][0].replace(/\by\b/, 'yyyy');

```